point_test.go

```diff
@@ -219,3 +220,18 @@ func TestPointXYZM(t *testing.T) {
 		})
 	}
 }
+
+func TestPointBuffer(t *testing.T) {
+
+	pointToTest := NewPoint(XY).MustSetCoords([]float64{0, 0})
+
+	bufferedPolygon, err := pointToTest.Buffer(8, 1)
+
+	if err != nil {
+		t.Error(err)
+	}
+
+	expectedFlatCoords := []float64{1, 0, math.Sqrt(2) / 2, math.Sqrt(2) / 2, 0, 1, -math.Sqrt(2) / 2, math.Sqrt(2) / 2, -1, 0, -math.Sqrt(2) / 2, -math.Sqrt(2) / 2, 0, -1, math.Sqrt(2) / 2, -math.Sqrt(2) / 2, 0, 1}
+
+	assert.InDeltaSlice(t, expectedFlatCoords, bufferedPolygon.FlatCoords(), 0.0001)
```

Comment:
The relatively large tolerated error here (0.0001) is likely due to the error accumulation mentioned above. How small can you make this tolerated error?

Reply:
hey, the smallest tolerated error that works is with this test is 1.e-15,
but changing the code above does not improve.